app/components/peabody/Tutorial.tsx

```diff
@@ -0,0 +1,58 @@
+import { useContext, useState } from "react";
+import { ScrollytellContext } from "~/scrollytellContext";
+import type { HighlightedElement } from "~/components/peabody/peabodyUtils";
+import { numberRange } from "~/utils";
+import TutorialYearSquare from "./TutorialYearSquare";
+import TutorialEventSquare from "./TutorialEventSquare";
+import eventsData from "~/data/peabody/1600sEvents.json";
+import TutorialKey from "./TutorialKey";
+
+export default function Tutorial() {
```

Comment:
Let's rename this to `TutorialPeabodySquare` to align with the other peabody square components